src/native.js

```diff
@@ -764,6 +752,29 @@ function toLegacyResponse(ortbResponse, ortbRequest) {
       legacyResponse[PREBID_NATIVE_DATA_KEYS_TO_ORTB_INVERSE[NATIVE_ASSET_TYPES_INVERSE[requestAsset.data.type]]] = asset.data.value;
     }
   }
+
+  // Handle trackers
+  legacyResponse.impressionTrackers = [];
+  let jsTrackers = [];
+
+  if (ortbRequest?.imptrackers) {
+    legacyResponse.impressionTrackers.push(ortbRequest.imptrackers);
```

Comment:
I think this should be `push(...ortbRequest.imptrackers)`. `imptrackers` [should be an array of URLs](https://www.iab.com/wp-content/uploads/2018/03/OpenRTB-Native-Ads-Specification-Final-1.2.pdf).